cards.js

```diff
@@ -7,51 +7,51 @@ let cardsJSON = `[
 		"deck": [
 			{
 				"question": "God said 'let there be light' on which of the first six days?",
-				"questionTags": [ "day" ],
+				"questionTags": [ "day", "<=6" ],
```

Comment:
what is "<=6" ?

Reply:
It's a tag that I made up meaning "a number less than or equal to six". It's just for testing purposes, though, so don't worry about it too much. I expect we'll just replace all the current tags after some planning, anyway.